doc/source/scalability/innodb_io.rst

```diff
@@ -76,10 +76,14 @@ The following values are allowed:
     use O_DIRECT to open the data files and fsync() system call to flush both the data and log files.
 
   * ``O_DIRECT_NO_FSYNC``:
-    use O_DIRECT to open the data files but don't use ``fsync()`` system call to flush both the data and log files. This option isn't suitable for *XFS* file system.
+    use O_DIRECT to open the data files and doublewrite file, but does not
+    use ``fsync()`` system call to flush both the data files, log files, and
+    parallel doublewrite file. This option isn't suitable for *XFS* file system.
 
   * ``ALL_O_DIRECT``: 
-    use O_DIRECT to open both data and log files, and use ``fsync()`` to flush the data files but not the log files. This option is recommended when |InnoDB| log files are big (more than 8GB), otherwise there might be even a performance degradation. **Note**: When using this option on *ext4* filesystem variable :variable:`innodb_log_block_size` should be set to 4096 (default log-block-size in *ext4*) in order to avoid the ``unaligned AIO/DIO`` warnings.
+    use O_DIRECT to open data files, log files, and parallel doublewrite file
+    and use ``fsync()`` to flush the data files but not the log files or 
+    doublewrite file. This option is recommended when |InnoDB| log files are big (more than 8GB), otherwise there might be even a performance degradation. **Note**: When using this option on *ext4* filesystem variable :variable:`innodb_log_block_size` should be set to 4096 (default log-block-size in *ext4*) in order to avoid the ``unaligned AIO/DIO`` warnings.
```

Comment:
```suggestion
    parallel doublewrite files. This option is recommended when |InnoDB| log files are big (more than 8GB), otherwise there might be even a performance degradation. **Note**: When using this option on *ext4* filesystem variable :variable:`innodb_log_block_size` should be set to 4096 (default log-block-size in *ext4*) in order to avoid the ``unaligned AIO/DIO`` warnings.
```

Reply:
updated.